src/components/utilities/requests.jsx

```diff
@@ -10,12 +10,12 @@ export const fetchAllArticles = async (topic, sortBy, orderBy) => {
       params: {
         topic,
         sort_by: sortBy,
-        order: orderBy
+        order: orderBy,
```

Comment:
There's nothing stopping you from just calling these `order` and `sort_by` when you invoke the function - then you wouldn't need different key:value pairs.